UI/BuildDesignatorWnd.cpp

```diff
@@ -1051,7 +1051,8 @@ void BuildDesignatorWnd::CenterOnBuild(int queue_idx, bool open) {
             MapWnd* map = ClientUI::GetClientUI()->GetMapWnd();
             map->CenterOnObject(system_id);
             if (open) {
-                SelectSystem(system_id);
+                HumanClientApp::GetApp()->GetClientUI().GetMapWnd()->SelectSystem(system_id);
+                //SelectSystem(system_id);
```

Comment:
@Dilvish-fo the fix works for me.

@adrianbroher, my references to the review in commit messages is intended
- to provide a link to the full discussion of the change.
- to acknowledge and credit the work of the reviewers, who are not committing the code so that it is clear in the commit log who found the bug/suggested the fix and did the work.

While we are using githib the references are obvious.

I realize the github will not last forever.  And when it is gone, if we choose not to archive these discussions are not archived then those references will be dangling.